scripts/constants.sh

```diff
@@ -22,3 +22,12 @@ relayer_dockerhub_repo=${DOCKER_REPO:-"awm-relayer"}
 current_branch=$(git symbolic-ref -q --short HEAD || git describe --tags --exact-match || true)
 
 git_commit=${RELAYER_COMMIT:-$( git rev-list -1 HEAD )}
+
+# Set the CGO flags to use the portable version of BLST
+#
+# We use "export" here instead of just setting a bash variable because we need
+# to pass this flag to all child processes spawned by the shell.
+export CGO_CFLAGS="-O -D__BLST_PORTABLE__"
```

Comment:
do we need this for teleporter as well?

Reply:
The teleporter repo itself doesn't have any Go applications that use BLST. Those should just be avalanchego, subnet-evm, and the relayer.